### PR TITLE
add option: restrict rocksdb log file num

### PR DESCRIPTION
**Pull request by DorianZheng**
## What have you changed? (mandatory)
Add an option to DBConfig & RaftDBConfig to restrict the info log file num

## What are the type of the changes? (mandatory)
- Improvement

## How has this PR been tested? (mandatory)
Write a test case

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
No

## Does this PR affect tidb-ansible update? (mandatory)
No

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -394,6 +394,7 @@ pub struct DbConfig {
     pub compaction_readahead_size: ReadableSize,
     pub info_log_max_size: ReadableSize,
     pub info_log_roll_time: ReadableDuration,
+    pub info_log_keep_log_file_num: u64,
```
Comment (huachaohuang) [inline src/config.rs]: I think we can default to 10 * 1GB log files.

File: tests/config/mod.rs [hunk 2]
```diff
@@ -171,6 +171,7 @@ fn test_serde_custom_tikv_config() {
         compaction_readahead_size: ReadableSize::kb(1),
         info_log_max_size: ReadableSize::kb(1),
         info_log_roll_time: ReadableDuration::secs(12),
+        info_log_keep_log_file_num: 1000,
```
Comment (huachaohuang) [inline tests/config/mod.rs]: You may want to use a different value than the default for the test.

File: src/config.rs (hunk 1 above)
Comment (DorianZheng) [reply src/config.rs]: ok

File: tests/config/mod.rs (hunk 2 above)
Comment (DorianZheng) [reply tests/config/mod.rs]: ok

File: etc/config-template.toml [hunk 3]
```diff
@@ -304,6 +304,30 @@
...
+# is larger than `max_log_file_size`, a new info log file will be created.
+# If max_log_file_size == 0, all logs will be written to one log file.
+# Default: 1GB
+# info-log-max-size = "1GB"
+
+# Time for the Rocksdb info log file to roll (in seconds).
+# If specified with non-zero value, log file will be rolled
+# if it has been active longer than `log_file_time_to_roll`.
+# Default: 0 (disabled)
+# info-log-roll-time = "1s"
```
Comment (huachaohuang) [inline etc/config-template.toml]: It should be the same as the default value.

**Comment by breeswish** [comment]
LGTM. Please also remember to file a PR in tidb-ansible repository, like [this](https://github.com/pingcap/tidb-ansible/pull/402).

**Pull request merged**